### ISSUE TITLE
[Feature Request] Enhanced Debt management

### ISSUE THREAD

**Issue by Xaver106** [mtotschnig/MyExpenses#1011]
This might be similar to #890

Instead of managing debts as either a separate Account or on the Debt screen, I think it makes sense to add another transaction type. Let's say you go get food with some friends but are the only one with cash on hand. You will pay for the meal and your friends will pay it back. What you will currently do in the app is create a transaction (normal or split) and then either create an account for each user or add debts in the Parties menu. 
Instead, It makes sense to add the debt directly to the transaction. This way when looking through the debts you get a lot more context from where the debt comes from and what you are owed, or what you owe. Clearing debt could then either be done by simply clearing it in the transaction, or by adding another transaction which marks the debt as clear.

It is important though, that you can close a debt with a different amount than it was opened. This way when paying back you can account for rounding up/down.

I hope this makes sense.

**Comment by mtotschnig** [comment]
@Xaver106 I am sorry, I was not able to react to this idea earlier. Could you share, why you think, it is no longer relevant?

**Comment by Xaver106** [comment]
Looks like I mixed something up, this was not what I wanted to close. Sorry for that

**Comment by mtotschnig** [comment]
@Xaver106 If the app would allow to create a new debt directly from the transaction screen, i.e. the new debt object in the database would have an amount of 0, and the transaction would be linked to it, thus setting its current value. Would this allow your use case to be fulfilled more easily?

**Comment by Xaver106** [comment]
Yes, linking debt with transactions would be great. Open a debt with a transaction and closing with another would be very helpful.
